.github/workflows/release.yml

```diff
@@ -1,10 +1,12 @@
 name: Release Airflow Images
 on:
+  schedule:  # triggers every midnight
+    - cron: '0 0 * * *' 
   push:
     branches:
       - master
     tags:
-      - "*" # triggers only if push new tag version
+      - "*" # triggers on a push event
```

Comment:
Is the push section still needed when on schedule?

Reply:
@mukrishn

Reply:
yes if any one makes a blocker fix and want to be reflected immediately, like the recent kube-burner version bump made many CI runs to fail.